release/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "agent-release",
+  "version": "1.0.0",
+  "description": "Flow for releasing agent",
+  "main": "mkrelease.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Nit: I'd probably cut this, but maybe have a release command so you can just say `npm release` rather than explicitly calling `node mkrelease.js` - I think that's more canonical node

Reply:
Could also have a couple different commands for different arg patterns

Reply:
I see how to make it:
```
npm run release
```
Is that what you mean?

I think we could actually move the dev.sh and. the other supporting scripts into here. We would need to rename the directory though.